specification/security/resource-manager/Microsoft.Security/preview/2023-02-01-preview/sqlVulnerabilityAssessmentsScanOperations.json

```diff
@@ -175,7 +175,7 @@
         ]
       }
     },
-    "ScanProperties": {
+    "ScanPropertiesV2": {
```

Comment:
need to be a new api-version for contract changes.

Reply:
Can you explain your comment. I am creating a new API version due to an update in the contract.
What is the action needed here?
Thanks!

Reply:
Sorry, I didn't see that. usually, you copy from old api-version and commit to create a base line, and then change to new api-version, so reviewer can see the diff.